tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -253,6 +253,12 @@ namespace.gluster:
         tags:
           help: optional parameters
           type: Dict
+        current_job:
```

Comment:
why are these new attributes added to NativeEvent object? It should be for the Volume object

Reply:
oh thats a mistake. will correct.